src/test/qunit.js

```diff
@@ -4,6 +4,8 @@ import moment from '../moment';
 
 export var test = QUnit.test;
 
+export var expect = QUnit.expect;
```

Comment:
Well, I noticed that assert.expect fails if it's actually executed. The [assert.expect](https://github.com/moment/moment/pull/2723#discussion-diff-44740979L301) that's in the codebase already doesn't get executed when you run the tests because it's protected by that if statement

![image](https://cloud.githubusercontent.com/assets/148787/11166334/ec3bde8e-8b93-11e5-8341-2b8f0857eb2d.png)